sentences/ru/_common.yaml

```diff
@@ -103,6 +103,8 @@ expansion_rules:
   set: (сдела(й|ть)|установи[ть]|постав(ь|ить)|поменя(й|ть)|измени[ть]|выстав(ь|ить)|переключи[ть]|переве(сти|ди)|увелич(ь|ить)|уменьш(и|ить)|подн(ими|ять)|опусти[ть]|прибав(ь|ить)|отн(ими|ять)|повыс(ь|ить)|пониз(ь|ить))
   close: (закр(ой|ыть)|прикр(ой|ыть)|опусти[ть])
   open: (откр(ой|ыть)|подн(ими|ять))
+  what_is: "како(е|й)|какие|какая|каков[о|a]|[на]сколько|когда"
+  what_is_the_class_of_name: "<what_is> [показани(е|я)|значени(е|я)|уровен(и|ь)] <class> [показывает[ся]|отображает[ся]] [у|на] [датчик(а|е)|сенсор(а|е)] <name> [<area>]"
```

Comment:
That part should be done in the separate PR, since it's not related to the PR subject.